syntax = "proto3";

package fraud_detection;

service FraudDetection {
    rpc CheckFraud (FraudCheckRequest) returns (FraudCheckResponse);
}

message FraudCheckRequest {
    string order_id = 1;
    double amount = 2;
}

message FraudCheckResponse {
    bool is_fraudulent = 1;
    string message = 2;
}